.github/workflows/push_dist_to_npm.yml

```diff
@@ -0,0 +1,31 @@
+name: Push dist to npm
+on: push
+
+jobs:
+  publish:
+    if: github.repository_owner == 'graphhopper'
+    runs-on: ubuntu-22.04
+    environment: npm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          registry-url: https://registry.npmjs.org/
+          node-version: lts/gallium
+
+      - name: Build
+
+        run: |
+          npm version --git-tag-version=false 0.0.0-$(git rev-parse HEAD)
+          # limit distribution to the dist folder (just for this build)
+          node -e "const packageJson=require('./package.json'); packageJson.files=['dist/']; require('fs').writeFileSync('package.json', JSON.stringify(packageJson, null, 4));"
```

Comment:
This build only releases the bundled/minified production app, not all the source code.